templates/text-embeddings/README.md

```diff
@@ -0,0 +1,100 @@
+# Computing Text Embeddings with Ray
+The Python file `ray_embeddings.py` is used to generate text embeddings using a pre-trained HuggingFace model. We also provide `ray_embeddings_notebook.ipynb` which has the same code logic, but in notebook form, in case you want to tinker and customize the logic.
+The high-level steps involved are:
+
+0. Read in raw data from user-provided input parquet file paths.
+1. Preprocess the raw input text by chunking to rows of 512 words.
+2. Compute embeddings using a pre-trained HuggingFace model (default model is [`thenlper/gte-large`](https://huggingface.co/thenlper/gte-large)).
+3. Write computed embeddings to parquet files in the user-provided output path.
+
+![Embeddings Computation Overview](diagram.jpg "Embeddings Computation Overview")
+
+## Running the code
+The provided script will work as-is; no need to modify anything the underlying script, unless you want to customize the logic to meet your specific requirements.
+
+Sample usage:
+```
+# Generate emeddings for a sample single-file parquet file containing 10 rows from the `falcon-refinedweb` HuggingFace dataset, using 1 GPU.
+python ray_embeddings.py \
```

Comment:
where is the python file?

Reply:
It's in the docker image used for the cluster environment. We want to keep this code private, so it is not included in this public repo.

Reply:
can you create a wrapper python file here which takes params and invokes proprietary script in docker image. Seems odd to ask users to directly invoke our python file in docker image

Reply:
yes, the relevant files (python file, ipynb, sample files, etc) are included in the `default` directory.

Reply:
Awesome thanks for confirming